UnityProject/Assets/Scripts/UI/Items/PDA/GUI_PDASettingMenu.cs

```diff
@@ -19,7 +19,7 @@ public void OnPageActivated()
 
 		public void SetRingtone(string ringtone)
 		{
-			if (ringtone == default) return;
+			if (ringtone == "") return;
```

Comment:
I thought ``default`` for strings was ""

Reply:
another way to check it would be
```cs
if (ringtone.IsNullOrEmpty()) return;
```

Reply:
It wasn't working as "== default", updated to use IsNullorEmpty

Reply:
Yeah so it turns out `default` for string is actually `null`, not an empty string

Reply:
:eyes: